src/main/resources/petriNets/engine-processes/preference_filter_item.xml

```diff
@@ -330,6 +362,9 @@
 		<i18nString name="allowed_roles_desc">Liste der Rollen, die diesen Menüeintrag anzeigen dürfen.</i18nString>
 		<i18nString name="banned_roles">Verbotene Rollen</i18nString>
 		<i18nString name="banned_roles_desc">Liste der Rollen, die diesen Menüeintrag nicht anzeigen dürfen.</i18nString>
+		<i18nString name="create_case_button_title">Schaltflächentitel "Neuer Fall"</i18nString>
+		<i18nString name="create_case_button_icon">Kennung des Schaltflächensymbols "Neuer Fall"</i18nString>
```

Comment:
`Kennung` might not be the correct translation. I believe we allow the users to specify icons in some process allready. It would be nice, if these translations matched each other (were consistent).